controller/ampachecontroller.php

```diff
@@ -501,4 +502,26 @@ protected function get_cover() {
 			return $r;
 		}
 	}
+
+	/**
+	 * this is not a part of ampache specification
+	 * This is an exposed API endpoint.
+	 * Client uses settings#addUserKey instead
+	 * @NoAdminRequired
+	 * @NoCSRFRequired
+	 * @CORS
```

Comment:
I would propose to not expose another API endpoint, but simply add the annotations to the existing one in https://github.com/owncloud/music/blob/acfe12afcd05447fa0dfc698b5ff5c7f930ef552/controller/settingcontroller.php#L73

Then you also don't need to add any route ;)